cli_ftd/user/udp_communication/udp_comm.c

```diff
@@ -40,50 +47,89 @@ static void get_sensors_data(char *);
 
 /* UDP APIs */
 static void udp__comm_init(otInstance *aInstance);
-static void udp__comm_send(otInstance *aInstance);
+static void udp__comm_send(otInstance *aInstance, char *);
```

Comment:
Probably just add char *udp_final_message, if you can

Reply:
Sure